### ISSUE TITLE
MDDropdown menu showing no content in  android

### ISSUE THREAD

**Issue by osama-ammar** [kivymd/KivyMD#942]
### Description of the Bug

MDDropdown menu is not showing no content in  android , but working in linux.
It was working well in android many weeks ago ,but not working now in android

### Code and Logs
'''
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivy/uix'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivy/uix/behaviors'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivy/uix/recycleview'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/fonts'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/images'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/stiffscroll'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/toast'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/toast/androidtoast'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/toast/kivytoast'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/tools'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/tools/packaging'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/tools/packaging/pyinstaller'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/uix'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/uix/behaviors'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd/utils'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/kivymd-0.104.2.dev0.dist-info'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/pkg_resources'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/pkg_resources/_vendor'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/pkg_resources/_vendor/packaging'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/pkg_resources/extern'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/pyjnius-1.2.1-py3.8.egg-info'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/setuptools'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/setuptools/_vendor'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/setuptools/_vendor/packaging'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/setuptools/command'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/setuptools/extern'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/setuptools-40.9.0-py3.8.egg-info'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/usr'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/usr/local'...
Listing '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/_python_bundle/_python_bundle/site-packages/usr/local/bin'...
[INFO]:    Detected highest available build tools version to be 31.0.0-rc2
[DEBUG]:   -> running gradlew assembleDebug
[DEBUG]:   	Starting a Gradle Daemon (subsequent builds will be faster)
[DEBUG]:   	
[DEBUG]:   	> Task :compileDebugJavaWithJavac
[DEBUG]:   	Note: Some input files use or override a deprecated API.
[DEBUG]:   	Note: Recompile with -Xlint:deprecation for details.
[DEBUG]:   	Note: Some input files use unchecked or unsafe operations.
[DEBUG]:   	Note: Recompile with -Xlint:unchecked for details.
[DEBUG]:   	
[DEBUG]:   	> Task :transformNativeLibsWithStripDebugSymbolForDebug
[DEBUG]:   	/home/osama/.buildozer/android/platform/android-ndk-r19c/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/bin/arm-linux-androideabi-strip:/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/build/intermediates/transforms/mergeJniLibs/debug/0/lib/armeabi-v7a/gdb.setup: File format not recognized
[DEBUG]:   	
[DEBUG]:   	Unable to strip library '1' due to error /home/osama/.buildozer/android/platform/android-ndk-r19c/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/bin/arm-linux-androideabi-strip returned from '/media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/build/intermediates/transforms/mergeJniLibs/debug/0/lib/armeabi-v7a/gdb.setup', packaging it as is.
[DEBUG]:   	
[DEBUG]:   	Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
[DEBUG]:   	Use '--warning-mode all' to show the individual deprecation warnings.
[DEBUG]:   	See https://docs.gradle.org/6.4.1/userguide/command_line_interface.html#sec:command_line_warnings
[DEBUG]:   	
[DEBUG]:   	BUILD SUCCESSFUL in 36s
[DEBUG]:   	27 actionable tasks: 27 executed

[INFO]:    <- directory context /media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/python-for-android
[INFO]:    Of the existing distributions, the following meet the given requirements:
[INFO]:    	Radiosh: min API 21, includes recipes (freetype, hostpython3, jpeg, libffi, openssl, png, sdl2_image, sdl2_mixer, sdl2_ttf, sqlite3, python3, sdl2, setuptools, pillow, six, pyjnius, android, kivy, https://github.com/kivymd/kivymd/archive/master.zip, certifi), built for archs (armeabi-v7a)
[INFO]:    Radiosh has compatible recipes, using this one
[INFO]:    # Copying android package to current directory
[INFO]:    # Android package filename not found in build output. Guessing...
[INFO]:    # Found android package file: /media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/build/outputs/apk/debug/Radiosh__armeabi-v7a-debug.apk
[INFO]:    # Add version number to android package
[INFO]:    # Android package renamed to Radiosh__armeabi-v7a-debug-2-.apk
[DEBUG]:   -> running cp /media/osama/Learn/programming/kivy/git_test/radiosh/.buildozer/android/platform/build-armeabi-v7a/dists/Radiosh__armeabi-v7a/build/outputs/apk/debug/Radiosh__armeabi-v7a-debug.apk Radiosh__armeabi-v7a-debug-2-.apk
WARNING: Received a --sdk argument, but this argument is deprecated and does nothing.
No setup.py/pyproject.toml used, copying full private data into .apk.
Applying Java source code patches...
Applying patch: src/patches/SDLActivity.java.patch
# Android packaging done!
# APK Radiosh-2-armeabi-v7a-debug.apk available in the bin directory
# Run '/home/osama/.buildozer/android/platform/android-sdk/platform-tools/adb devices'
# Cwd None
List of devices attached
EEFU6PMZCEVKMZHM	device

# Deploy on EEFU6PMZCEVKMZHM
# Run '/home/osama/.buildozer/android/platform/android-sdk/platform-tools/adb install -r "/media/osama/Learn/programming/kivy/git_test/radiosh/bin/Radiosh-2-armeabi-v7a-debug.apk"'
# Cwd /home/osama/.buildozer/android/platform
Performing Streamed Install
Success
# Application pushed.
# Run on EEFU6PMZCEVKMZHM
# Run '/home/osama/.buildozer/android/platform/android-sdk/platform-tools/adb shell am start -n org.radiosh.radiosh/org.kivy.android.PythonActivity -a org.kivy.android.PythonActivity'
# Cwd /home/osama/.buildozer/android/platform
Starting: Intent { act=org.kivy.android.PythonActivity cmp=org.radiosh.radiosh/org.kivy.android.PythonActivity }

'''

### Screenshots

https://photos.app.goo.gl/c2fhGTK6VUa1Tgaw6

### Versions
buildozer requirements = python3,kivy==2.0.0,https://github.com/kivymd/KivyMD/archive/master.zip,sdl2_ttf==2.0.15,pillow

operating system:
* OS: Ubunto
* Python: 3.7.9
* Kivy: 2.0.0
* KivyMD: 0.104.2.dev0

**Comment by HeaTTheatR** [comment]
@osama-ammar Attach a minimal code...

**Comment by HeaTTheatR** [comment]
@osama-ammar 
```python
    from kivy.lang import Builder
    from kivy.metrics import dp
    from kivy.properties import StringProperty

    from kivymd.uix.list import OneLineIconListItem
    from kivymd.app import MDApp
    from kivymd.uix.menu import MDDropdownMenu

    KV = '''
    <IconListItem>

        IconLeftWidget:
            icon: root.icon


    MDScreen

        MDDropDownItem:
            id: drop_item
            pos_hint: {'center_x': .5, 'center_y': .5}
            text: 'Item 0'
            on_release: app.menu.open()
    '''


    class IconListItem(OneLineIconListItem):
        icon = StringProperty()


    class Test(MDApp):
        def __init__(self, **kwargs):
            super().__init__(**kwargs)
            self.screen = Builder.load_string(KV)
            menu_items = [
                {
                    "viewclass": "IconListItem",
                    "icon": "git",
                    "text": f"Item {i}",
                    "height": dp(56),
                    "on_release": lambda x=f"Item {i}": self.set_item(x),
                } for i in range(5)
            ]
            self.menu = MDDropdownMenu(
                caller=self.screen.ids.drop_item,
                items=menu_items,
                position="center",
                width_mult=4,
            )
            self.menu.bind()

        def set_item(self, text_item):
            self.screen.ids.drop_item.set_item(text_item)
            self.menu.dismiss()

        def build(self):
            return self.screen


    Test().run()
```

**Comment by osama-ammar** [comment]
A sample from my code
`
from kivy.lang import Builder
from kivymd.app import MDApp
import datetime
from kivy.properties import ObjectProperty
from kivymd.uix.tab import MDTabsBase
from kivymd.uix.menu import MDDropdownMenu
from kivy.uix.boxlayout import BoxLayout


root = Builder.load_string('''

<DLP>:

  BoxLayout:
    id:DLP
    orientation:'vertical'

    GridLayout:
      cols:3
      rows:1
      spacing:[100]
      padding:[20,20,20,20]
      size_hint:(1,.6)       

      MDLabel:
      
        text: "DLP :"
        pos_hint: {'x':.2, 'y':.5}  

        size_hint:(.05,.001)    

        font_size: "16sp"
            
            
            
      MDTextFieldRound:
        id: input_DLP
        height: "30dp"
        mode: "fill"
        pos_hint: {'x':0.0, 'y':.5}              
        size_hint:(.05,.5)  


      MDLabel:
      
        text: " mGy*cm "
        pos_hint: {'x':1, 'y':1}  
        size_hint:(.05,.001)    
        font_size: "14sp"




    FloatLayout:
      spacing:[20]
      padding:[40,40,20,20] 

      
          
      MDRoundFlatButton:
        text: "Target Organ"
        id:organs
        pos_hint: {'x':.15, 'y':.5}
        on_release: root.organs_menu.open()
        padding:[40,40,40,40]


      MDRoundFlatButton:
        text: "   Age   "
        id:age
        pos_hint: {'x':.6, 'y':.5}
        on_release: root.age_menu.open()
        padding:[40,40,40,40]



    FloatLayout:
      spacing:[20]
      padding:[20,20,20,20] 

      
          
      MDRoundFlatButton:
        text: "calculate"
        id:calculate

        pos_hint: {'x':.35, 'y':.8}
        on_release: root.calculate_DLP()
        padding:[40,40,40,40]





    FloatLayout:
      spacing:[20]
      padding:[20,20,20,20] 

      
          
      MDLabel:
        id:ct_dose
        text: "Dose is 0000 msv"
        pos_hint: {'x':.3, 'y':.8}  

        font_size: "16sp"





    FloatLayout:
      spacing:[20]
      padding:[30,30,30,30] 

         
      MDLabel:
    
        text: "Reference: ICRP Publication 103.The 2007 Recommendations of the International Commission on Radiological Protection‏."
        pos_hint: {'x':.05, 'y':.9}  
        font_size: "10sp"




''')



class DLP(MDTabsBase,BoxLayout):

    def __init__(self,*args, **kwargs):
        super(DLP,self).__init__(*args, **kwargs)
        

        menu_items = [{"text":"Head"},{"text":"Neck"}, {"text":"Chest"}, {"text":"Abdomen"}, {"text":"Pelvis"}]
        self.organs_menu = MDDropdownMenu(caller=self.ids.organs ,items=menu_items, width_mult=3)
        self.organs_menu.bind(on_release=self.set_item_organ)
        
        
        age_items = [{"text":"Adult"},{"text":"10 years"}, {"text":"5 years"}, {"text":"1 year"}, {"text":"New Born"}]
        self.age_menu = MDDropdownMenu(caller=self.ids.age ,items=age_items, width_mult=3)            

        self.age_menu.bind(on_release=self.set_item_age)


    def set_item_organ(self, instance_menu, instance_menu_item):
        self.ids.organs.text=instance_menu_item.text
        self.organs_menu.dismiss()
 
    def set_item_age(self, instance_menu, instance_menu_item):
        self.ids.age.text=instance_menu_item.text
        self.age_menu.dismiss()





    def calculate_DLP(self,*args):

        Head=[	0.0019,	0.0027,	0.0035,	0.0054,	0.0087]
        Neck=[	0.0052,	0.0094,	0.0121,	0.0168,	0.021]
        Chest=[	0.0146,	0.0237,	0.0323,	0.0482,	0.0739]
        Abdomen=[	0.0153,	0.0249,	0.0357,	0.053,	0.0841]
        Pelvis=[	0.0129,	0.0219,	0.03,	0.0446,	0.0701]


        organ_list=[]
        age=0
        
        try:   
           dlp=float(self.ids.input_DLP.text)
        except ValueError:   
            dlp=0.0        
        
        
        

        
        
        if  self.ids.organs.text=='Head':
            organ_list=Head
            
        elif  self.ids.organs.text=='Neck':
            organ_list=Neck
            
        elif  self.ids.organs.text=='Chest':
            organ_list=Chest 
            
        elif  self.ids.organs.text=='Abdomen':
            organ_list=Abdomen
            
        elif  self.ids.organs.text=='Pelvis':
            organ_list=Pelvis
       
            
            
        print (self.ids.organs.text)
      #activity should be in GBq
        age_dict={"Adult":0,"10 years":1, "5 years":2, "1 year":3, "New Born":4}          
        for i in age_dict.keys():				
            if  self.ids.age.text==i:
              age=int(age_dict[i])
              



        try:
            print (age)
            print (organ_list)
            k=organ_list[age]
            dose=dlp*k
            self.ids.ct_dose.text= 'Dose  :  %f   msv '%dose 
            
        except IndexError:   
            self.ids.ct_dose.text= 'select organ & age '    
      

class MyApp3(MDApp):

    def build(self):
        self.theme_cls.primary_palette = "Blue"
        self.theme_cls.accent_palette = 'Green'

        return DLP()


`

**Comment by osama-ammar** [comment]
Dear  @HeaTTheatR  ther's no problem in the code , it's alraedy working in computer the problem is in android,Could you please clarify , why you put the above code?

**Comment by HeaTTheatR** [comment]
@osama-ammar 
https://discord.com/channels/566880874789076992/618108815774056488/816265169880875018
![Снимок экрана 2021-04-03 в 21 33 45](https://user-images.githubusercontent.com/16930280/113488062-6d1c3c80-94c4-11eb-8f42-9aeb689bc084.png)

**Comment by mandomal** [comment]
I understand this was closed a while back but it looks like there was no resolution. I have a similar issue and also on all examples I've tried (including the one posted above) "on_release" method seems broken when built on android.

Also, from the example posted here the item does show text but no icon when built on android. Using a viewclass of OneLineListItem does work however.

Aside from this example, I cannot get other examples to properly show any text. The number of items do populate which tells me that the number of dictionaries are processed in an android apk, but the dictionary keys and values are either tossed or just not handled at all.
